docs/how-to-guides/deployment-options.md

```diff
@@ -38,6 +38,66 @@ To learn more about the various options available to customize your service, and
 
 #### Option 2: Deploy a single Bicep file locally
```

Comment:
Can you also help fill out basics of pre-reqs, e.g., az login, az set subscription